cpp/include/cudf/datetime.hpp

```diff
@@ -178,7 +178,8 @@ std::unique_ptr<cudf::column> day_of_year(
  * @endcode
 
  * @param[in] timestamps cudf::column_view of timestamp type.
- * @param[in] months cudf::column_view of integer type containing the number of months to add.
+ * @param[in] months cudf::column_view of integer type containing the number of
+ * months to add.
```

Comment:
This is the only change to this file. Is this change required?

Reply:
nope